.github/workflows/lock.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
IIUC this defaults to `${{ github.token }}` so it's not necessary to make it explicit.

Reply:
I don't have access to the github secrets so I'm not sure what's available.  I see that in release.yml we use `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}`.  Are both of `secrets.GITHUB_TOKEN` and `github.token` set to the same value?

Reply:
I think older versions of lock-threads required this setting but it should not be needed now

Reply:
> Are both of `secrets.GITHUB_TOKEN` and `github.token` set to the same value?

Yep, from [the docs](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context):
> github.token | string | A token to authenticate on behalf of the GitHub App installed on your repository. This is functionally equivalent to the GITHUB_TOKEN secret.